app/models/manageiq/providers/redfish/physical_infra_manager/event_catcher.rb

```diff
@@ -0,0 +1,13 @@
+module ManageIQ::Providers::Redfish
+  class PhysicalInfraManager::EventCatcher < ::MiqEventCatcher
```

Comment:
You shouldn't need the `MiqEventCatcher` alias here if we're adding this as new, just using `ManageIQ::Providers::BaseManager::EventCatcher` should be fine.  We usually add an alias just to make refactoring easier but new code shouldn't have to use the alias.